BOCCHI/Plugin.cs

```diff
@@ -62,6 +63,7 @@ private void SetupLanguage(IDalamudPluginInterface plugin)
             ClientLanguage.French => "fr",
             ClientLanguage.German => "de",
             ClientLanguage.Japanese => "jp",
+            ClientLanguage.ChineseSimplified => "zh",
```

Comment:
I'd have to look into this https://github.com/goatcorp/Dalamud/blob/master/Dalamud/Game/ClientLanguage.cs